test/pdfboxing/form_test.clj

```diff
@@ -42,12 +42,12 @@
 
 (deftest populating-fields
   (testing "Error handling for non simple fonts"
-    (is (= "Error: can't add non-simple fonts, this is a constraint of PDFBox."
-           (set-fields "test/pdfs/interactiveform.pdf"
-                       "test/pdfs/filled-in.pdf"
-                       {"Name_Last" "Last",
-                        "Name_First" "First",
-                        "Name_Middle" "Middle"}))))
+    (is (thrown? java.io.IOException
+                 (set-fields "test/pdfs/interactiveform.pdf"
+                             "test/pdfs/filled-in.pdf"
+                             {"Name_Last" "Last",
+                              "Name_First" "First",
+                              "Name_Middle" "Middle"}))))
```

Comment:
That's the default formatting with Emacs and clojure-mode 4.1.0. If you'd like to deviate I can push a commit with the whitespace removed, it's up to you. Thanks for responding so quickly.